rootfs/etc/nginx/template/nginx.tmpl

```diff
@@ -930,6 +930,7 @@ stream {
 
             proxy_pass            http://upstream_balancer;
             log_by_lua_block {
```

Comment:
My current issue is that I'm struggling to understand nginx phases, so while this works in the `log_by_lua_block` I'm not sure if this is the right directive. Any guidance here would be appreciated. I tried using `rewrite_by_lua_block` but this phase is to early

Reply:
@rikatz @strongjz Any ideas?

Reply:
seems like `access_by_lua_block` works